send2trash/__main__.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import print_function
+
+import sys
+
+from argparse import ArgumentParser
+from send2trash import send2trash
+
+
+def main():
+    parser = ArgumentParser(description='Tool to send files to trash')
+    parser.add_argument('files', nargs='+')
+    parser.add_argument('-v', '--verbose', action='store_true', help='Print deleted files')
+    args = parser.parse_args()
+    
+    exit_code = 0
+    for filename in args.files:
+        try:
+            send2trash(filename)
+            if args.verbose:
+                print("Trashed '" + filename + "'")
```

Comment:
A Simple aesthetic improvement, if you may:
```suggestion
                print('Trashed «', filename, '»')
```